protocol/x/accountplus/authenticator/authentication_request.go

```diff
@@ -131,26 +123,12 @@ func extractExplicitTxData(tx sdk.Tx, signerData authsigning.SignerData) (Explic
 		return ExplicitTxData{}, errorsmod.Wrap(sdkerrors.ErrInvalidType, "failed to cast tx to TxWithMemo")
 	}
 
-	// Encode messages as Anys and manually convert them to a struct we can serialize to json for cosmwasm.
-	txMsgs := tx.GetMsgs()
-	msgs := make([]LocalAny, len(txMsgs))
-	for i, txMsg := range txMsgs {
-		encodedMsg, err := types.NewAnyWithValue(txMsg)
-		if err != nil {
-			return ExplicitTxData{}, errorsmod.Wrap(err, "failed to encode msg")
-		}
-		msgs[i] = LocalAny{
-			TypeURL: encodedMsg.TypeUrl,
-			Value:   encodedMsg.Value,
-		}
-	}
-
```

Comment:
removing this for simplicity. don't really need to serialize for cosmwasm for now